runtime/runtime/src/lib.rs

```diff
@@ -1328,7 +1328,7 @@ impl Runtime {
                 &mut stats,
                 epoch_info_provider,
             );
-            tracing::debug!(target: "runtime", node_counter = ?state_update.trie().get_trie_nodes_count());
+            tracing::trace!(target: "runtime", node_counter = ?state_update.trie().get_trie_nodes_count(), "process_receipt done");
```

Comment:
…and if the interesting part from these logs are the `node_counter` delta, then the span should have a KV for that, rather than leaving it up to user to figure out which two events are matched and do the computation themselves.

But it is also fine to leave as proposed if you decide you don't want to do this for us.

Reply:
Well, that makes me want to make it a single log line with both values even more :)